bats/tests/helpers/paths.bash

```diff
@@ -1,17 +1,64 @@
+# PATH_BATS_ROOT and PATH_BATS_HELPERS are already set by load.bash
+
+PATH_REPO_ROOT=$(
+    # We are running with `set -o errexit`, so no need for explicit error checking
+    # shellcheck disable=SC2164
+    cd "$PATH_BATS_ROOT/.."
+    pwd
+)
+
+set_path_resources() {
+    local system=$1
+    local user=$2
+    local dist=$3
+    local subdir=$4
+
+    if [ -z "$RD_LOCATION" ]; then
+        if [ -d "$system" ]; then
+            RD_LOCATION=system
+        elif [ -d "$user" ]; then
+            RD_LOCATION=user
+        elif [ -d "$PATH_REPO_ROOT/pkg/rancher-desktop" ]; then
+            # dist and npm only work when inside a repo clone
+            if [ -d "$dist" ]; then
```

Comment:
This would be (slightly) clearer to say `elif [ -f "$dist" ]; then … elif [ -d "$PATH_REPO_ROOT/pkg/rancher-desktop" ]; then…`
If we're not inside a repo checkout, `$dist` wouldn't exist.

Reply:
I don't understand what you are saying. Can you use the "suggestion" feature to show your proposed replacement?

> If we're not inside a repo checkout, $dist wouldn't exist.

Yes, and that's why I check first that we are inside a repo checkout before testing `$dist`. So we are in agreement on this part, but why would you then change the code order?